repos/spack_repo/builtin/packages/palace/package.py

```diff
@@ -3,92 +3,107 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack_repo.builtin.build_systems.cmake import CMakePackage
+from spack_repo.builtin.build_systems.cuda import CudaPackage
+from spack_repo.builtin.build_systems.rocm import ROCmPackage
```

Comment:
In addition to the spack style questions, I don't quite understand why the older package API is being used in our local testing, and that we didn't catch these required changes sooner.

See [slack thread](https://spackpm.slack.com/archives/C08Q62S7XEX/p1754419018258109)